include/zephyr/bluetooth/conn.h

```diff
@@ -1396,6 +1396,20 @@ enum bt_security_err {
 	BT_SECURITY_ERR_UNSPECIFIED,
 };
 
+struct bt_conn_le_cs_procedure_enable_complete {
```

Comment:
For this file and cs.h I think it would be good to document what each parameter means and what values are allowed.
Otherwise an app developer needs to know the core spec/have the core spec open to be able to use this